pallets/runtime/develop/src/runtime.rs

```diff
@@ -295,7 +295,7 @@ pallet_staking_reward_curve::build! {
 
 parameter_types! {
     // Six sessions in an era (24 hours).
-    pub const SessionsPerEra: sp_staking::SessionIndex = 6;
+    pub const SessionsPerEra: sp_staking::SessionIndex = 3;
```

Comment:
Do we want this in this PR? or It is just for testing?

Reply:
Yep - was intentional ;-)